packages/big-design/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,104 @@
+import { CheckIcon } from '@bigcommerce/big-design-icons';
+import React, { HTMLAttributes, memo, useEffect, useState } from 'react';
+
+import {
+  StyledDash,
+  StyledLight,
+  StyledLightText,
+  StyledMobileLight,
+  StyledMobileLightText,
+  StyledStep,
+  StyledStepper,
+  StyledText,
+} from './styled';
+
+export interface StepProps {
+  number: number;
+  text?: string;
+  state?: 'pending' | 'current' | 'done';
+  isLast?: boolean;
+}
+
+export interface StepperProps extends HTMLAttributes<HTMLDivElement> {
+  steps: Array<string>;
+  currentStep: number;
+}
+
+export interface LightProps {
+  active?: boolean;
+}
+export interface DashProps {
+  active?: boolean;
+}
+export interface TextProps {
+  active?: boolean;
+}
+
+export const Step: React.FC<StepProps> = ({ number, text, state, isLast }) => {
```

Comment:
Move this component into its own file: `.../src/components/Stepper/Step/Step.tsx`